daemon/daemon_main.go

```diff
@@ -335,6 +335,9 @@ func init() {
 	flags.Bool(option.EnableK8sExternalIPs, defaults.EnableK8sExternalIPs, fmt.Sprintf("Enable k8s externalIPs feature (Only enabled in conjunction with %s)", option.EnableNodePort))
 	option.BindEnv(option.EnableK8sExternalIPs)
 
+	flags.Bool(option.K8sEnableEndpointSlice, defaults.K8sEnableEndpointSlice, "Enables k8s EndpointSlice feature into Cilium if the k8s cluster supports it")
```

Comment:
Not a native speaker here, but `enables... feature in/for`?

Reply:
I don't know, `in/for` does not look correct :-/

Reply:
`in` looks right to me. The feature is already inside Cilium (well, when we merge this PR anyway). Not worth blocking merge on this though.

Reply:
Fixed, I thought you were saying something like `in the k8s cluster supports it` or `for the k8s cluster supports it`, which was not making sense to me.